packages/web/src/components/tipping/feed-tip-tile/FeedTipTile.tsx

```diff
@@ -257,10 +313,14 @@ export const FeedTipTile = () => {
           receiver={usersMap[tipToDisplay.receiver_id]}
         />
       </div>
-      <div className={styles.buttons}>
+      <div className={styles.buttons} ref={sendTipRef}>
         <SendTipButton
           user={usersMap[tipToDisplay.receiver_id]}
-          hideName={useShortButtonFormat}
+          hideName={useShortButtonFormat.current}
```

Comment:
Instead of all the JS logic, can we add CSS media queries for various screen size breakpoints? That's typically how responsive ui is done afaik

Reply:
I think that was how we were doing it before but that's where the problem came from. The resize really depends on the width of the other contents of the tile, not the screen size. There's probably a way we can determine whether to resize by incorporating the screen size, but I don't think that'll save us from having to do these calculations?

Reply:
Hmm I see, we're not resizing based on window width alone, but based on widths of the button + the other content being >= the container?

Reply:
Yep. Tho I could use those numbers to calculate the window size at which we'd need to resize if that's preferred.

Reply:
I think I get it and what you have seems right...

Sorry for my confusion, I'm ramping up to speed here - there's two things happening from what I see:
1) The button is changing at the component level based on how much space it has - this is what you've solved here with measuring the size of the two elements and the container and passing a prop to the button. This makes sense because the goal is to change the button based on how long the names are, not how big the screen is - otherwise we could use CSS media queries here to hide the "to (username)" elements.
2) The content is overflowing when it shouldn't on small screens even after the smaller button - this is a "responsive" bug and is going to need some `flex-shrink`, `flex-wrap`, or CSS media queries. This is out of scope for the PR